internal/contour/endpointstranslator_test.go

```diff
@@ -418,21 +582,24 @@ func TestEndpointsTranslatorScaleToZeroEndpoints(t *testing.T) {
 
 	// Assert endpoint was added
 	want := []proto.Message{
-		envoy.ClusterLoadAssignment("default/simple", envoy.SocketAddress("192.168.183.24", 8080)),
+		&v2.ClusterLoadAssignment{
+			ClusterName: "default/simple",
+			Endpoints:   envoy.WeightedEndpoints(1, envoy.SocketAddress("192.168.183.24", 8080)),
+		},
 	}
-	got := et.Contents()
 
-	assert.Equal(t, want, got)
+	protobuf.ExpectEqual(t, want, et.Contents())
```

Comment:
I don't think using `assert` or `ExpectEqual` here is correct - if there is a failure here, there's no point running the rest of the test. I think it should be `RequireEqual`.